gtk4/src/expression.rs

```diff
@@ -182,6 +182,56 @@ impl glib::value::ToValueOptional for Expression {
     }
 }
 
+/// Trait containing convenience methods in creating [`gtk::PropertyExpression`] that
+/// looks up a property of a [`glib::Object`].
+pub trait PropExpr {
+    /// Create an expression looking up an object's property
+    fn property_expression(&self, property_name: &str) -> Expression;
+
+    /// Create an expression looking up an object's property with a weak reference.
+    fn weak_property_expression(&self, property_name: &str) -> Expression;
+}
+
+impl<T: IsA<glib::Object>> PropExpr for T {
```

Comment:
I'm not sure about the trait name. `GObjectPropertyExpressionExt` maybe?

Reply:
I think that's clearer